src/components/Header/Header.js

```diff
@@ -1,13 +1,16 @@
 import './Header.css';
 import { header } from '../../config/header';
 import Button from '../Button/Button';
-import HeaderNav from '../HeaderNav/HeaderNav';
 import { htmlIds } from '../../config/nav-link';
 
 function Header() {
   return (
     <header className="header" id={htmlIds.brandId}>
-      <HeaderNav />
+      <div className="header__left-column">
```

Comment:
should this be re-written as `block__element_modifier`, eg. `header__column_left` & `header__column_right` ?